lib/SPIRV/SPIRVInternal.h

```diff
@@ -50,6 +50,7 @@
 #include "llvm/IR/Attributes.h"
 #include "llvm/IR/Constants.h"
 #include "llvm/IR/Instructions.h"
+#include "llvm/IR/IntrinsicInst.h"
```

Comment:
We can avoid this include in the .h file; please use a forward declaration here and include the header only in .cpp files.

Reply:
Done